BootloaderCommonPkg/Library/TpmLib/TpmLib.inf

```diff
@@ -37,6 +37,7 @@
 [Packages]
   MdePkg/MdePkg.dec
   BootloaderCommonPkg/BootloaderCommonPkg.dec
+  BootloaderCorePkg/BootloaderCorePkg.dec
```

Comment:
BootloaderCommonPkg should not depend on BootloaderCorePkg (CorePkg could depend on CommonPkg).